searx/static/themes/simple/src/js/main/keyboard.js

```diff
@@ -117,65 +105,80 @@ searxng.ready(function () {
       des: 'open the result in a new tab',
       cat: 'Results'
     },
-  }
-
-  // these bindings are enabled by user preferences
-  var vimKeys = {
-    'b': {
-      key: 'b',
-      fun: scrollPage(-window.innerHeight),
-      des: 'scroll one page up',
-      cat: 'Navigation'
-    },
-    'f': {
-      key: 'f',
-      fun: scrollPage(window.innerHeight),
-      des: 'scroll one page down',
-      cat: 'Navigation'
-    },
-    'u': {
-      key: 'u',
-      fun: scrollPage(-window.innerHeight / 2),
-      des: 'scroll half a page up',
-      cat: 'Navigation'
-    },
-    'd': {
-      key: 'd',
-      fun: scrollPage(window.innerHeight / 2),
-      des: 'scroll half a page down',
-      cat: 'Navigation'
-    },
-    'g': {
-      key: 'g',
-      fun: scrollPageTo(-document.body.scrollHeight, 'top'),
-      des: 'scroll to the top of the page',
-      cat: 'Navigation'
-    },
-    'v': {
-      key: 'v',
-      fun: scrollPageTo(document.body.scrollHeight, 'bottom'),
-      des: 'scroll to the bottom of the page',
-      cat: 'Navigation'
-    },
-    'k': {
-      key: 'k',
-      fun: highlightResult('up'),
-      des: 'select previous search result',
-      cat: 'Results'
-    },
-    'j': {
-      key: 'j',
-      fun: highlightResult('down'),
-      des: 'select next search result',
-      cat: 'Results'
-    },
   };
-
-  if (searxng.settings.hotkeys) {
-    // To add Vim-like key bindings, merge the 'vimKeys' into 'keyBindings'.
-    Object.assign(keyBindings, vimKeys);
+  var keyBindingLayouts = {
+
+    "default": Object.assign(
+      { /* SearXNG layout */
+        'ArrowLeft': {
+          key: '←',
+          fun: highlightResult('up'),
+          des: 'to select previous search result',
```

Comment:
```suggestion
          des: 'select previous search result',
```

Reply:
Sorry, I'm an idiot .. thanks for double check / [fixed](https://github.com/searxng/searxng/compare/5c68aeee7311496b1d971f356a99e0c7fbbc0538..f1d9b9680b9391c03923919278d4cf87582e9a65) :+1:

Reply:
> Sorry, I'm an idiot ..
>
Certainly not :)
> thanks for double check / [fixed](https://github.com/searxng/searxng/compare/5c68aeee7311496b1d971f356a99e0c7fbbc0538..f1d9b9680b9391c03923919278d4cf87582e9a65) 👍
>
You're always welcome!